Jenkinsfile

```diff
@@ -90,50 +103,24 @@ timestamps {
               archiveArtifacts 'coverage-report/lcov-report/index.html'
             }
           }
-        }
-
-        stage('Sonar') {
-          onPR {
-            sh """
-              yarn sonar-scanner -- \
-              -Dsonar.analysis.mode=preview \
-              -Dsonar.host.url=$SONARQUBE_URL
-            """
-          }
-
-          onMaster {
+          stage('Sonar') {
             sh "yarn sonar-scanner -- -Dsonar.host.url=$SONARQUBE_URL"
           }
-        }
-
-        stage('Package application (Docker)') {
-          citizenFrontendVersion = dockerImage imageName: 'cmc/citizen-frontend'
-        }
 
-        stage('Package application (RPM)') {
-          citizenFrontendRPMVersion = packager.nodeRPM('citizen-frontend')
-          version = "{citizen_frontend_buildnumber: ${citizenFrontendRPMVersion}}"
+          stage('Package application (RPM)') {
```

Comment:
@rlewan, @timja-kainos , @yarpo what your opinion on building RPM's on PR branch as well?

Reply:
How likely is it that RPM build will fail? Not too likely in my opinion and a possibility of master build failing because of that is a risk I'm willing to take 👍

Reply:
Agree with you @rlewan. I am a bit in between but more keen to do it only on master.

Reply:
If we see this is causing trouble we can always bring building RPMs on PR back.

Reply:
In the many months doing this I've only seen it break twice for legitimate reasons, and a few times for jenkins being full of disk space. Not worth doing it on PRs imo